src/components/Article/SocialButtons.tsx

```diff
@@ -42,6 +51,28 @@ export const SocialButtons = ({
         </button>
         <p className="text-gray-100 text-sm">{article.comments_count}</p>
       </div>
+      <div className="flex flex-col text-center gap-1 w-10">
+        <button onClick={() => {
+          const modal = document.getElementById('my_modal_1') as HTMLDialogElement;
+          modal.showModal();
+        }}>
+          <DotsThreeOutline className="text-gray-100 w-10 h-10" weight="fill" />
+        </button>
+        <dialog id="my_modal_1" className="modal">
+          <div className="modal-box">
+            <div className="flex flex-col w-1/2 items-center justify-center m-auto">
+              <button className="btn bg-accent m-2" onClick={() => copyToClipboard(article)}>Copy link</button>
+              <button className="btn bg-accent m-2 block md:hidden" onClick={() => navigator.share({ title: article.title, url: article.url })}>Share to</button>
```

Comment:
Ficou ótimo cara!
Pesquisei um pouco aqui e descobri que alguns navegadores também possuem esse share nativo, achei bem curioso, nesse caso, acho que seria interessante renderizarmos ele sempre que possível, que tal?
```suggestion
              <button className={`btn bg-accent m-2 block ${navigator.share ? "" : "hidden"`} onClick={() => navigator.share({ title: article.title, url: article.url })}>Share to</button>
```

Reply:
Fiz a alteração, mas usei um typeof para verificar se o navigator.share é uma função. Só verificando se navigator.share não é null resultaria sempre em true.